### DISCUSSION TITLE
How To Add TrainData in YoloV5

### DISCUSSION THREAD

**Discussion by icon-global** [ultralytics/yolov5#6689] [Q&A]
I use YoloV5.

First I train [Box] in Train Data.
[Box] Image is 300 files.
So Train Time need 4 Hours.

Teach me How to Add New Data to [best.pt] update.
1) How to [Box] data  Add [Canned juice] Images.
2) How to [Box] data  Add New [Box] Images.

**Comment by glenn-jocher** [comment]
@icon-global see our community post on this topic:
https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets
